shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/controllers/BridgeExecutorController.java

```diff
@@ -97,12 +101,16 @@ public UpdateControl<BridgeExecutor> reconcile(BridgeExecutor bridgeExecutor, Co
             LOGGER.info("Executor deployment BridgeProcessor: '{}' in namespace '{}' is NOT ready", bridgeExecutor.getMetadata().getName(),
                     bridgeExecutor.getMetadata().getNamespace());
 
-            bridgeExecutor.getStatus().setConditionsFromDeployment(deployment);
+            setStatusFromDeployment(bridgeExecutor, deployment);
```

Comment:
I moved the method `setConditionsFromDeployment(deployment)` from `CustomResourceStatus` as it was only used here.